eos-initramfs-efi-depends

```diff
@@ -1,3 +1,4 @@
+eos-plymouth-theme
```

Comment:
Please also explicitly add `plymouth` as that's what the dracut module actually uses. In fact, looking at `/etc/dracut.conf.d/endless.conf`, I see other things in `dracutmodules` that are not explicitly called out here. E.g., `systemd` and `e2fsprogs`.

Reply:
systemd is already in there, but I was counting on dracut's existing dependency on e2fsprogs to fill that need, much in the same way I was counting on eos-plymouth-theme to bring in plymouth.

I've force pushed changes adding both plymouth and e2fsprogs to be more direct about dependencies.  I hope I've got them all now. :/